integration/createdeletebeforeactive_test.go

```diff
@@ -0,0 +1,83 @@
+// +build integration
+
+package integration_test
+
+import (
+	"fmt"
+
+	awseks "github.com/aws/aws-sdk-go/service/eks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/testutils/aws"
+	. "github.com/weaveworks/eksctl/pkg/testutils/matchers"
+)
+
+var _ = Describe("(Integration) Create & Delete before Active", func() {
+	const (
+		initNG = "ng-0"
+		testNG = "ng-1"
+	)
+
+	Describe("when creating a cluster with 1 node", func() {
+		var clName string
+		Context("for deleting a creating cluster", func() {
+			It("should run eksctl and not wait for it to finish", func() {
```

Comment:
i don't see any assertions, should this be in an initialization block instead? (I'm not familiar with ginkgo enough to know where to put that)

Reply:
I re-worked this.  The check for an error happens within the helper function `eksctlStart`.